src/ORM/Table.php

```diff
@@ -1130,7 +1149,7 @@ public function updateAll($fields, $conditions)
      *   use null to get a validator.
      * @return \Cake\Validation\Validator
      */
-    public function validator($name = 'default', Validator $validator = null)
+    public function validator($name = self::DEFAULT_VALIDATOR, Validator $validator = null)
```

Comment:
You might want `static` here so people can override the constant name if they see a need to.